app/components/img-selector.js

```diff
@@ -15,6 +16,9 @@ export default Ember.Component.extend({
                 }
             };
             reader.readAsDataURL(e.target.files[0]);
+        },
+        clickInput: function() {
+            this.$().find('.img-selector-input').click();
```

Comment:
1. Should this be `Ember.$`?
2. Given that the parent selector is "everything", is `.find` necessary? Can we just use `Ember.$('.img-selector-input')` instead?
3. Based on code usages, this seems to assume that only one such element will be present. Should `img-selector-input` be an ID instead of a class?